taipy/core/data/_data_repository.py

```diff
@@ -118,10 +118,10 @@ def _from_model(self, model: _DataNodeModel):
             id=model.id,
             name=model.name,
             parent_id=model.parent_id,
-            last_edition_date=datetime.fromisoformat(model.last_edition_date) if model.last_edition_date else None,
+            last_edit_date=datetime.fromisoformat(model.last_edit_date) if model.last_edit_date else None,
```

Comment:
What about something like : 
```suggestion
            last_edit_date = datetime.fromisoformat(model.last_edit_date) if model.last_edit_date else datetime.fromisoformat(model.last_edition_date) if model.last_edition_date else None,
```
So we accept to read the last_edition_date field but we don't write it. Using such a mechanism, the old JSON files could be still compatible with the new field name. For that, we need to keep temporarily the 2 names in the dataNodeModel class.

Reply:
Since last_edition_date is no longer in the model, model.last_edition_date will throw AttributeError. Do you think we should also store it in the model?

Reply:
Is there anyway we can read them without storing in the model?

Reply:
Can we save the model if it's the old version?

Reply:
The principle I propose is to be able to read both old and new models, but we only write the new model.